WordPress/Classes/ViewRelated/Gutenberg/Processors/GutenbergMediaFilesUploadProcessor.swift

```diff
@@ -35,7 +35,11 @@ class GutenbergMediaFilesUploadProcessor: Processor {
             }
 
             do {
-                try FileManager.default.moveItem(at: URL(string: mediaFile.url)!, to: newURL)
+                if let mediaURL = URL(string: mediaFile.url) {
+                    try FileManager.default.moveItem(at: mediaURL, to: newURL)
+                } else {
+                    DDLogError("No Media File URL was present. This is Stories error and should be investigated.")
```

Comment:
`This is Stories error and should be investigated` > `This is a Stories error and should be investigated`